README.md

```diff
@@ -1,10 +1,10 @@
 # HPE OneView SDK for GoLang
 
 ## Build Status 
-OV Version | 6.20 | 6.10 | 6.00 | 5.60 | 5.50 | 5.40 | 
-| ------------- |:-------------:| -------------:| -------------:| -------------:| -------------:|-------------:|
-SDK Version/Tag | [v6.2.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.2.0) | [v6.1.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.1.0)  | [v6.0.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.0.0) | [v1.8.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.8.0) | [v1.7.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.7.0) | [v1.6.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.6.0) | [v1.5.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.5.0) | 
-Build Status | [![Build status](https://github.com/HewlettPackard/oneview-golang/actions/workflows/ci.yml/badge.svg)](https://github.com/HewlettPackard/oneview-golang/actions/runs/998462546)| [![Build status](https://github.com/HewlettPackard/oneview-golang/actions/workflows/ci.yml/badge.svg)](https://github.com/HewlettPackard/oneview-golang/actions/runs/789405541)| [![Build status](https://github.com/HewlettPackard/oneview-golang/actions/workflows/ci.yml/badge.svg)](https://github.com/HewlettPackard/oneview-golang/actions/runs/638584153)| [![Build status](https://github.com/HewlettPackard/oneview-golang/actions/workflows/ci.yml/badge.svg)](https://github.com/HewlettPackard/oneview-golang/actions/runs/568946462)| [![Build status](https://github.com/HewlettPackard/oneview-golang/actions/workflows/ci.yml/badge.svg)](https://github.com/HewlettPackard/oneview-golang/actions/runs/347277855)| [![Build status](https://github.com/HewlettPackard/oneview-golang/actions/workflows/ci.yml/badge.svg)](https://github.com/HewlettPackard/oneview-golang/actions/runs/249321396)|
+OV Version | 6.30 | 6.20 | 6.10 | 6.00 | 5.60 | 5.50 | 5.40 | 
+| ------------- | ------------- |:-------------:| -------------:| -------------:| -------------:| -------------:|-------------:|
```

Comment:
please have consistent patterns. Add :-------------: instead of -------------

Reply:
done